scrapers/nh/actions.py

```diff
@@ -0,0 +1,35 @@
+from utils.actions import Rule, BaseCategorizer
+
+# These are regex patterns that map to action categories.
+_categorizer_rules = (
+    Rule(r"Minority Committee Report", None),  # avoid calling these passage
+    Rule(r"Ought to Pass", ["passage"]),
+    Rule(r"Passed by Third Reading", ["reading-3", "passage"]),
+    Rule(r".*Ought to Pass", ["committee-passage-favorable"]),
+    Rule(r".*Introduced(.*) and (R|r)eferred", ["introduction", "referral-committee"]),
+    Rule(r"Proposed(.*) Amendment", "amendment-introduction"),
+    Rule(r"Amendment .* Adopted", "amendment-passage"),
+    Rule(r"Amendment .* Failed", "amendment-failure"),
+    Rule(r"Signed", "executive-signature"),
+    Rule(r"Vetoed", "executive-veto"),
+    Rule(r"^Introduced", "introduction"),
+    Rule(r"Enrolled Adopted", "enrolled"),
+    Rule(r"^Inexpedient", "failure"),
+    Rule(r"Withdraws Floor Amendment", "amendment-withdrawal"),
+    Rule(r"Refer to Interim Study", "referral"),
+    Rule(r"^(?!.*Committee Report).*[Rr]eferred [Tt]o.*$", "referral-committee"),
+    Rule(r"Lay.*on Table.*$", "deferral"),
+    Rule(
+        r"^(?!.*Minority).*Committee Report: Inexpedient to Legislate",
+        "committee-failure",
+    ),
+)
+
+
+class Categorizer(BaseCategorizer):
```

Comment:
I see this boilerplate in a few different scrapers. Is it possible to simplify this? Maybe make this part of the default object so people can just say `from utils.actions import Categorizer`? Or is that worth the effort?

Reply:
Just for my understanding, are you suggesting we move this chunk of code:
```
class Categorizer(BaseCategorizer):
    rules = rules

    def categorize(self, text):
        attrs = BaseCategorizer.categorize(self, text)
        return attrs
```
to utils.actions so  we can directly import `Categorizer`? I like that idea but the `rules` definition would still need to happen in each jurisdiction's `actions.py`.  I think we would still need to have
```
class Categorizer(BaseCategorizer):
     rules = rules
```
in every actions.py?

Reply:
I missed that bit. Probably best to keep things this way and just document _how/why_ we import classifiers.